webapp/redux/middleware/plotMiddleware.js

```diff
@@ -106,6 +106,7 @@ export default (store) => (next) => (action) => {
     showSpinner("Loading plot...");
     return plotFigure(widget.id, method.plotMethod, method.plotType, uri, store.getState().general.theme)
       .then((data) => {
+        data = '<div style="bk {margin: 0 auto!important;} div {overflow:hidden;} frame {overflow:hidden;}>'+data+'</div>';
```

Comment:
I don't understand this change, there is a div inside the div tag, it seems also that we are trying to customise all the plots here when instead for the bokeh plots we agreed on removing the overflow:hidden added previously (set in the plotViewer.js).

Reply:
the rest of the feedback is here https://metacell.atlassian.net/browse/NETPYNE-81?focusedCommentId=10029